### ISSUE TITLE
problem about training the train.py 

### ISSUE THREAD

**Issue by zxm115** [twke18/Adaptive_Affinity_Fields#5]
![screenshot_2018-09-29 tensorboard_segloss](https://user-images.githubusercontent.com/25972022/46239245-bdccef00-c3c8-11e8-9070-e97a83d546c1.png)
![screenshot_2018-09-29 tensorboard_reducedloss](https://user-images.githubusercontent.com/25972022/46239248-c3c2d000-c3c8-11e8-8566-fe6dfd461359.png)
![screenshot_2018-09-29 tensorboard_l2loss](https://user-images.githubusercontent.com/25972022/46239252-cf15fb80-c3c8-11e8-95b9-dee7551603cd.png)
Sorry, I am currently loading the pre-trained model according to the parameter settings in the paper and the sample file named train_pspnet.sh in the bashscripts directory, also using the PASCAL VOC data set, but the same problem still occurs, the loss function can not be reduced, resulting in the prediction result of the verification data set is not correct,which it can't display the content, I would like to ask you how to solve this situation, thank you for your help.

**Comment by sunkyya** [comment]
@zxm115 I have the same problem with you,have you solved it? I used deeplabV3+，added aaf loss to the original loss,but failed. The original loss can not converge and the result is nonsence.
